net/net-working.sh

```diff
@@ -0,0 +1,1181 @@
+#!/usr/bin/env bash
```

Comment:
was this file included intentionally? looks like some kind of temp/working copy of `net.sh`